exchange/targeting_test.go

```diff
@@ -566,11 +618,68 @@ var TargetingTests []TargetingTestData = []TargetingTestData{
 		},
 		TruncateTargetAttr: nil,
 	},
+	{
+		Description: "Cache and deal targeting test custom prefix",
+		TargetData: targetData{
+			priceGranularity:  lookupPriceGranularity("med"),
+			includeBidderKeys: true,
+			cacheHost:         "cache.prebid.com",
+			cachePath:         "cache",
+			prefix:            "blah",
```

Comment:
May you please change it to something more professional, like "test_prefix" or "custom_prefix"?